source/docs/reference/configurations.md

```diff
@@ -121,3 +121,28 @@ Available values are : "products", "categories" and "pages".
 ```js
 module.exports = ["products", "categories", "pages"];
 ```
+
+### `config/caching.js`
+
+Allows to define configurations related to dataloader caching and implementation
+
+<blockquote class="wip">
+**Work In Progress:** more details about specific configurations will be detailed as part of https://github.com/front-commerce/developers.front-commerce.com/issues/49. If you need it right away, please [contact us](mailto:contact@front-commerce.com). We will make sure to answer you in a timely manner.
+</blockquote>
+
+* `DEFAULT_MAX_BATCH_SIZE`: default batch size used for dataloaders (unless specified during instanciation). Default: 100
+* `redis`: redis strategy configuration
+
+```js
+module.exports = {
+  DEFAULT_MAX_BATCH_SIZE: 100,
+  redis: {
+    caches: "*", // or ["LoaderKeyA", "LoaderKeyB"]
+    disabled: ["CatalogPrice"],
+    config: {
+      // host: "redis"
```

Comment:
I think that this host might be more confusing than relevant. 127.0.0.1 seems enough here.